src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListCommand.cs

```diff
@@ -160,13 +162,15 @@ private void MenuItemCallback(object sender, EventArgs e)
                     break;
 
                 case UsefulResultCommandId:
+                    SarifErrorListItem currentItem = selectedItems.FirstOrDefault();
                     var feedback = new FeedbackModel(
-                        this.selectionService.SelectedItem.Rule.Id,
-                        this.selectionService.SelectedItem.Tool.Name,
-                        this.selectionService.SelectedItem.Tool?.Version,
-                        this.selectionService.SelectedItem.GetCodeSnippets(),
+                        currentItem.Rule.Id,
```

Comment:
>currentItem.Rule.Id, [](start = 24, length = 20)

if we select 2 results from different rules we would send  as one.

Reply:
Combined RuleId/ToolNames/Versions if multiple values